examples/distributions/mass_examples.py

```diff
@@ -0,0 +1,44 @@
+import matplotlib.pyplot as plt
+from pycbc.inference import distributions
+
+# Create a mass distribution object that is uniform between 0.5 and 1.5
+# solar masses.
+mass1_distribution = distributions.Uniform(mass1=(0.5, 1.5))
+# Take 100000 random variable samples from this uniform mass distribution.
+mass1_samples = mass1_distribution.rvs(size=1000000)
+
+# Create another mass distribution object that is Gaussian between 0.5 and
+# 1.5 solar masses with a mean of 1.2 solar masses and a standard deviation
+# of 0.15 solar masses. Gaussian takes the variance as an output so square
+# the standard deviation.
+variance = 0.15*0.15
+mass2_distribution = distributions.Gaussian(mass2=(0.5, 1.5), mass2_mean=1.2,
+                                            mass2_var=variance)
+
+# Take 100000 random variable samples from this gaussian mass distribution.
+mass2_samples = mass2_distribution.rvs(size=1000000)
+
+# We can make pairs of distributions together, instead of apart.
+two_mass_distributions = distributions.Uniform(mass3=(1.6, 3.0),
+                                               mass4=(1.6, 3.0))
+two_mass_samples = two_mass_distributions.rvs(size=1000000)
+
+# Choose 20 mass bins for the histogram subplots.
+n_bins = 20
+
+# Make some subplots
+fig, axes = plt.subplots(nrows=2, ncols=2)
+ax0, ax1, ax2, ax3, = axes.flat
+
+ax0.hist(mass1_samples['mass1'], bins = n_bins)
```

Comment:
It probably would've been good to plot the PDF over the samples like:
```
import matplotlib.pyplot as plt
import numpy
from pycbc.inference.distributions import Gaussian

# set random seed
numpy.random.seed(0)

# draw samples from Gaussian distribution
g = Gaussian(mass2=(0,10), mass2_mean=5, mass2_var=1)
samples = g.rvs(size=500000)

# Get PDF
mass2 = numpy.arange(0, 10, 0.1)
pdf = [g.pdf(mass2=i) for i in mass2]

# plot drawn samples and PDF
nbins = 50
plt.hist(samples["mass2"], nbins, normed=True, histtype="step")
plt.plot(mass2, pdf)
plt.ylabel("Probability Density Function")
plt.xlabel(r"Mass 2 ($M_{\odot}$)")
plt.show()
```
Though I can add that in a future PR.